RocketData.py

```diff
@@ -53,66 +54,82 @@
  7:"LANDED",
  8:"WINTER_CONTINGENCY"
 }
-
 class RocketData:
     def __init__(self):
         self.timeset = {}
         self.lasttime = 0
         self.highest_altitude = 0
+        self.sessionName = str(int(time.time()))
+        t = threading.Thread(target=self.timer)
+        t.start()
```

Comment:
```suggestion
        self.autosaveThread = threading.Thread(target=self.timer)
        self.autosaveThread.start()
```

It is usually a good idea to keep a reference to all threads.